Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -412,7 +409,7 @@
     <value>There is a problem</value>
   </data>
   <data name="Supplementary_Index_UnableToAuthenticate" xml:space="preserve">
-    <value>Unable to authenticate user, please retry</value>
+    <value>Unable to authenticate user, Try again</value>
```

Comment:
Checked with Jason, this one should be a full stop before the word "Try"
<img width="489" alt="{78F57E1C-3D2D-4E21-861A-6AF380DC813E}" src="https://github.com/user-attachments/assets/c5288183-5c27-4ad9-8633-a4698bd995e7" />
```suggestion
    <value>Unable to authenticate user. Try again</value>
```